test/node-quirks.test.js

```diff
@@ -66,6 +110,25 @@ describe('#fetch (Node-only quirks)', function () {
     });
 
     describe('`files` option', function () {
+      it('should support Buffers', async function () {
+        const owlbert = await fs.readFile(`${__dirname}/fixtures/owlbert.png`);
+
+        const res = await fetchHar(harExamples['multipart-data-dataurl'], {
+          files: {
+            'owlbert.png': owlbert,
+          },
+          multipartEncoder: FormDataEncoder,
+        }).then(r => r.json());
+
+        expect(res.files).to.deep.equal({
+          // This won't have `name=owlbert.png` in the data URL that comes back to us because we sent a raw file buffer.
+          foo: owlbertDataURL.replace('name=owlbert.png;', ''),
+        });
+
+        expect(res.headers['Content-Length']).to.equal('579');
```

Comment:
do we not have the same boundary length issues here as we did with the previous test?

Reply:
No because all Node tests use `formdata-node` so the boundary identifier is always the same.